osquery/tables/system/windows/cpu_info.cpp

```diff
@@ -0,0 +1,58 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/sql.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+#include "osquery/core/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+QueryData genCpuInfo(QueryContext& context) {
+  Row r;
+  QueryData results;
+
+  WmiRequest wmiSystemReq("SELECT * FROM Win32_Processor");
+  std::vector<WmiResultItem>& wmiResults = wmiSystemReq.results();
+  if (wmiResults.empty()) {
+    LOG(WARNING) << "Error retreiving information from WMI.";
```

Comment:
Is there a way we can get more specific here? Get an actual message. Also this should be an ERROR not WARNING since we're failing to get the data.

Reply:
Some of the other tables specify this condition as a WARNING so I wanted to follow suit there. I'm happy to switch this over though, if that's how we want to go forward with it.

Reply:
I agree with @fmanco here, but perhaps we should stick with consistency to just get this landed, and I can file a follow-up issue to A.) Draft better documentation for contributing around philosophy of logging (read - when to use WARNING, INFO, ERROR, and VLOG(1)) and B.) an issue to migrate the current WMI tables away from logging WARNING when they fail to get data.

Reply:
I agree with keeping consistency here.